composer.json

```diff
@@ -11,16 +11,16 @@
     ],
     "require": {
         "php": ">=5.4",
-        "symfony/symfony": "~2.6",
+        "symfony/symfony": "~3",
```

Comment:
Looking at the changes made, I think this bundle will work with Symfony versions 2.7, 2.8, too. So the rule should be something like `"symfony/symfony": "~2.6|~3.0"`